assets/packages/questions/upload/build/uploadquestion.js

```diff
@@ -726,7 +726,10 @@
         } // sending request
 
 
-        var iframe = this._createIframe();
+        var iframe = this._createIframe(); // Get response from iframe and fire onComplete event when ready
+
+
```

Comment:
One new-line too much?

Reply:
Actually, after reviewing, this wasn't me. It was `yarn`. This is a build file.

I added the comment here:
https://github.com/LimeSurvey/LimeSurvey/pull/1552/files#diff-a87413b370f86ff754a96b5051d4f9d7R661